examples/image_classification/ios/ImageClassifier/Helper/ImageClassifierHelper.swift

```diff
@@ -0,0 +1,125 @@
+    super.init()
+    guard let modelPath = model.modelPath else { return }
+    let imageClassifierOptions = ImageClassifierOptions()
+    imageClassifierOptions.runningMode = runningModel
+    imageClassifierOptions.maxResults = maxResults
+    imageClassifierOptions.scoreThreshold = scoreThreshold
+    imageClassifierOptions.baseOptions.modelAssetPath = modelPath
+    imageClassifierOptions.imageClassifierLiveStreamDelegate = runningModel == .liveStream ? self : nil
+    imageClassifier = try? ImageClassifier(options: imageClassifierOptions)
+    self.delegate = delegate
+  }
+
+  func classify(image: UIImage) -> ResultBundle? {
+    guard let mpImage = try? MPImage(uiImage: image) else { return nil }
+    do {
+      let startDate = Date()
+      guard let result = try imageClassifier?.classify(image: mpImage) else { return nil }
+      let inferenceTime = Date().timeIntervalSince(startDate) * 1000
+      return ResultBundle(inferenceTime: inferenceTime, imageClassifierResults: [result])
+    } catch {
+      print(error)
+      return nil
+    }
+  }
+
+  func classifyVideoFile(url: URL, inferenceIntervalMs: Double) async -> ResultBundle? {
+    guard let imageClassifier = imageClassifier else { return nil }
+    let startDate = Date()
+    let asset:AVAsset = AVAsset(url: url)
+    let generator = AVAssetImageGenerator(asset:asset)
+    generator.requestedTimeToleranceBefore = CMTimeMake(value: 1, timescale: 25)
+    generator.requestedTimeToleranceAfter = CMTimeMake(value: 1, timescale: 25)
+    generator.appliesPreferredTrackTransform = true
+    guard let videoDurationMs = try? await asset.load(.duration).seconds * 1000 else { return nil }
+    let frameCount = Int(videoDurationMs / inferenceIntervalMs)
+    var imageClassifierResults: [ImageClassifierResult?] = []
+    for i in 0..<frameCount {
+      let timestampMs = Int(inferenceIntervalMs) * i // ms
+      let image:CGImage?
+      do {
+        let time = CMTime(seconds: Double(timestampMs) / 1000, preferredTimescale: 600)
+        try image = generator.copyCGImage(at: time, actualTime:nil)
+      } catch {
+        print(error)
+         return nil
+      }
+      guard let image = image else { return nil }
+      let uiImage = UIImage(cgImage:image)
+      let result = try? imageClassifier.classify(videoFrame: MPImage(uiImage: uiImage), timestampInMilliseconds: timestampMs)
+      imageClassifierResults.append(result)
+    }
+    let inferenceTime = Date().timeIntervalSince(startDate) / Double(frameCount) * 1000
+    return ResultBundle(inferenceTime: inferenceTime, imageClassifierResults: imageClassifierResults)
+  }
+
+  func classifyAsync(videoFrame: CMSampleBuffer, orientation: UIDeviceOrientation, timeStamps: Int) {
+    var uiimageOrientation: UIImage.Orientation = .up
+    switch orientation {
+    case .landscapeLeft:
+      uiimageOrientation = .left
+    case .landscapeRight:
+      uiimageOrientation = .right
+    default:
```

Comment:
@khanhlvg @st-tuanmai 
For now, I think these values are correct. But when the new CocoaPods is released with the orientation fix, .left and .right need to be interchanged.

You can also handle .protraitUpsideDown if the app supports it.

Reply:
> @khanhlvg @st-tuanmai For now, I think these values are correct. But when the new CocoaPods is released with the orientation fix, I think .left and .right need to be interchanged.
> 
> You can also handle .protraitUpsideDown if the app supports it.

The app don't currently have the Upside down orientation enabled, so I don't think the handle protraitUpsideDown is needed.

Reply:
> @khanhlvg @st-tuanmai For now, I think these values are correct. But when the new CocoaPods is released with the orientation fix, I think .left and .right need to be interchanged.
> 
> You can also handle .protraitUpsideDown if the app supports it.

The app don't currently have the Upside down orientation enabled, so I don't think the handle protraitUpsideDown is needed.

Reply:
@khanhlvg @st-tuanmai I have a correction to make here. These values for orientation will work fine when the new version with the orientation fix is released. For now, to see the correct results, you should put .right when orientation is .landscapeLeft and .left when orientation is .landscapeRight

Reply:
You can follow the same in the object detector too.